### PR TITLE
Allow migration of detached volumes

### PR DESCRIPTION
**Pull request by Pearl1594**

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan package

**Review by davidjumani — APPROVED** [review]
LGTM! Thanks @Pearl1594 !
@rhtyd Good to go!

File: src/config/section/storage.js [hunk 1]
```diff
@@ -168,7 +168,7 @@ export default {
           label: 'label.migrate.volume',
           args: ['volumeid', 'storageid', 'livemigrate'],
           dataView: true,
-          show: (record, store) => { return record.state === 'Ready' && ['Admin'].includes(store.userInfo.roletype) && record.virtualmachineid },
+          show: (record, store) => { return record.state === 'Ready' && ['Admin'].includes(store.userInfo.roletype) },
```
Comment (rhtyd) [inline src/config/section/storage.js:171]: @Pearl1594 @davidjumani do we allow migration of attached VMs, for all hypervisors? What's the logic/check in old UI?

Comment (davidjumani) [reply src/config/section/storage.js:171]: @rhtyd 
```
if (jsonObj.type == "ROOT" || jsonObj.type == "DATADISK") {
  if (jsonObj.state == "Ready" && isAdmin() && jsonObj.virtualmachineid != null) {
    allowedActions.push("migrateVolume");
  }
}
```

**Pull request merged**